cmd/katib-controller/v1beta1/main.go

```diff
@@ -130,8 +133,13 @@ func main() {
 	}
 
 	log.Info("Setting up webhooks.")
-	if err := webhook.AddToManager(mgr, webhookPort); err != nil {
-		log.Error(err, "Unable to register webhooks to the manager")
+	webhookErrMsg := "Unable to register webhooks to the manager"
+	if initConfig.ControllerConfig.WebhookPort == nil {
+		log.Error(fmt.Errorf("invalid webhookPort in .init.controller.webhookPort: %v", initConfig.ControllerConfig), webhookErrMsg)
+		os.Exit(1)
+	}
```

Comment:
Do we need this check if we set default value for Webhook port if it is Null: https://github.com/kubeflow/katib/blob/42fe2783ed7124f7a3c6717eb902af0a1ef7abec/pkg/apis/config/v1beta1/defaults.go#L87

Reply:
It sounds more reasonable. This seems over check.

Reply:
done.